ado-manifests/azure-devops-extension-ppe.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "banner-settings-hub-ppe",
     "name": "Banner Settings (PPE)",
-    "version": "1.0.70",
+    "version": "1.0.71",

```

Comment:
1.1.0 ?

Reply:
We are not using these version numbers anymore. They will be overwritten by the release pipeline. This number needs to be here but it meaningless, so I think it is fine as is.